src/Mvc/Mvc.ApiExplorer/src/ApiResponseTypeProvider.cs

```diff
@@ -317,7 +318,8 @@ private static void CalculateResponseFormatForType(ApiResponseType apiResponse,
             return declaredReturnType;
         }
 
-        private static bool IsClientError(int statusCode)
+        // Shared with EndpointMetadataApiDescriptionProvider
+        internal static bool IsClientError(int statusCode)
```

Comment:
I don't see where this is still used outside of this class.

```suggestion
        private static bool IsClientError(int statusCode)
```